CONTRIBUTING.md

```diff
@@ -1,177 +1,141 @@
-
-Example:
-
-Short and descriptive example bug report title
-
-A summary of the issue and the browser/OS environment in which it occurs. If suitable, include the steps required to reproduce the bug.
-
-1. This is the first step
-2. This is the second step
-3. Further steps, etc.
-
-`<url>` - a link to the reduced test case
-
-Any other information you want to share that is relevant to the issue being reported. This might include the lines of code that you have identified as causing the bug, and potential solutions (and your opinions on their merits).
-
-### Issues bots
-
-[Hound](https://houndci.com) is a bot and part of the core team that hangs out in our GitHub issue tracker and automatically checks for style validation errors during pull requests. If it finds any errors, it will post a follow-up comment on the issue and point out the errors. Think of Hound as your Drill Sergeant. If this happens with a pull request you're submitting, please fix the errors and resubmit. Hound allows the core team to better review and maintain a clean codebase.
-
-[Travis](https://travis-ci.org/) runs our test suite on each check in and soon will even deploy each change that gets merged.
-
-## Feature requests
-
-Feature requests are welcome. But take a moment to find out whether your idea
-fits with the scope and aims of the project. It's up to *you* to make a strong
-case to convince the project's core team of the merits of this feature. Please provide as much detail and context as possible. Some things to consider:
-
-1. Search Issues for similar feature requests. It's possible somebody has
-already asked for this feature or provided a pull request that we're still
-discussing.
-
-2. Provide a clear and detailed explanation of the feature you want and why
-it's important to add. Keep in mind that we want features that will be useful
-to veterans learning to code. If we need to create a separate repo, we can do that too.
+### Ruby
+Ruby is an Object Oriented programming language Operation Code is written in. While you can install ruby directly to your machine it's highly recommended to use a version manager.
 
-3. If the feature is complex, consider writing some initial documentation for
-it. If we do end up accepting the feature it will need to be documented and
-this will also help us to understand it better ourselves.
+* [Install rvm](https://rvm.io/rvm/install)
+* Look [here](https://raw.githubusercontent.com/OperationCode/operationcode/master/.ruby-version) to find the version of ruby we're using
+* Install the version of ruby from the step above. eg `rvm install 2.2.3`
 
-Still want to contribute? Carry on.
+### git
+Git is a distributed version control system. This is how our code is stored and managed. 
 
-## Environment setup
-1. Install [latest Ruby](https://www.ruby-lang.org/en/downloads/)
-2. Install [latest Rails](http://docs.railsbridge.org/installfest/choose_your_operating_system)
```

Comment:
Agree, then point to it. 

> On Apr 2, 2016, at 10:06 AM, Rick <notifications@github.com> wrote:
> 
> In CONTRIBUTING.md:
> 
> >  
> > -## Environment setup
> > -1. Install [latest Ruby](https://www.ruby-lang.org/en/downloads/)
> > -2. Install [latest Rails](http://docs.railsbridge.org/installfest/choose_your_operating_system)
> Yes - bundle install should install rails for you so no need to install it.
> 
> The installfest docs do seem pretty good but don't include postgresql. Maybe it makes sense to do our app specific install stuff (psql, redis etc) then point people to that guide?
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
>